auslib/admin/views/forms.py

```diff
@@ -223,6 +224,14 @@ class ScheduledChangeExistingRuleForm(ScheduledChangeForm, EditRuleForm):
     rule_id = IntegerField('Rule ID', validators=[Required()])
 
 
+class ScheduledChangeDeleteRuleForm(ScheduledChangeForm):
+    """
+    ScheduledChangeDeletionForm includes all the PK columns and ScheduledChangeForm columns
+    """
```

Comment:
This comment needs an update - data_version is neither a ScheduledChangeForm column nor a PK.